.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "mocha": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
+  },
+  "rules": {
```

Comment:
How did these rules come about? Did you convert from JSLint, or is this some default or recommended set?

Reply:
These rules extends the default eslint recommended settings and also I imported the ones we were previously using on `jshint`